DNN Platform/Library/Prompt/CommandRepository.cs

```diff
@@ -16,16 +16,16 @@
 
 namespace DotNetNuke.Prompt
 {
-    public class CommandRepository : ServiceLocator<ICommandRepository, CommandRepository>, ICommandRepository
+    public class CommandRepository : ServiceLocator<IDnnCommandRepository, CommandRepository>, IDnnCommandRepository
```

Comment:
Similar concern here, as without us adjusting this, the ICommandRepository will not have a service implementation found

Reply:
Changing the service locator here would definitely break anyone using it.